apollo-testing-support/src/commonMain/kotlin/com/apollographql/apollo/testing/MockQuery.kt

```diff
@@ -14,6 +14,9 @@ class MockQuery : Query<MockQuery.Data, Operation.Variables> {
 
   override fun responseFieldMapper(): ResponseFieldMapper<Data> {
     return ResponseFieldMapper {
+      while (it.selectField(emptyArray()) != -1) {
+        // consume the json stream
+      }
```

Comment:
The more I think about it, the more I think these tests should be integration tests. We have a bunch of tests that use `MockQuery/Subscription` and that are very fragile to codegen changes